resolwe_bio/tools/cuffnorm_output_table_headers.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python2
+# pylint: disable=missing-docstring,invalid-name
+# XXX: Refactor to a comand line tool and remove pylint disable
+"""Rename cuffnorm output tables headers."""
+from __future__ import absolute_import, division, print_function
+
+import argparse
+from operator import itemgetter
+
+parser = argparse.ArgumentParser(description="Rename cuffnorm output tables headers")
+parser.add_argument('files', help="All .cxb files")
+parser.add_argument('replicates', help="Define replicates and groups")
+parser.add_argument('sample_names', help="Sample names")
+args = parser.parse_args()
+
+files = args.files.split(",")
+replicates = args.replicates.split(",")
+sample_names = args.sample_names.split(",")
+
+samples=[]
+for x in sorted(zip(files,replicates,sample_names), key=itemgetter(1)):
```

Comment:
Add a comment that this sort should be the same as in `cufflinks_sample_groups.py`. And also add comment there that sort must me synced with this one.